app/javascript/controllers/simple_tree_controller.js

```diff
@@ -7,18 +7,17 @@ export default class extends Controller {
   }
 
   connect () {
-    let activeElem = this.element.querySelector('a.active')
-    if (activeElem) {
-      this.element.scrollTo({
-        top: activeElem.offsetTop,
-        behavior: 'smooth'
-      });
-
-      if (this.autoClickValue) {
-        activeElem.click()
+    setTimeout(() => {
+      let activeElem = this.element.querySelector('.tree-link.active');
+      if (activeElem) {
+        activeElem.scrollIntoView({ behavior: 'smooth', block: 'center' });
```

Comment:
```
this.element.scrollTo({
        top: activeElem.offsetTop,
        behavior: 'smooth'
      });
```
was better, no ? because it scrolls only the tree block not "all the page + the tree block" ?

Reply:
activeElem.offsetTop is always equal to 0

Reply:
Maybe change it to something else, here the goal is to scroll only the tree block to the position of the active element. And don't move/scroll all the page.

Reply:
https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/61744974/25ed9e53-b9f9-4eb4-8e76-76cd559b10f8